core/components/list-item/list-is-tight.ts

```diff
@@ -1,5 +1,7 @@
+import Token from 'markdown-it/lib/token';
```

Comment:
lets make this `import type Token ...` , so that the core package can make it a devDependency.